qemu/tests/driver_in_use.py

```diff
@@ -66,6 +67,12 @@ def run_bg_test_simu(bg_stress_test):
                 utils_test.run_virt_sub_test, (test, params, env),
                 {"sub_type": bg_stress_test})
             stress_thread.start()
+
+        if params.get("set_bg_stress_flag", "no") == "yes":
+            if not utils_misc.wait_for(lambda: env.get("memory_stable"),
+                                       240, 0, 1):
```

Comment:
why set timeout to 240?

Reply:
for windows, need 180s to get memory to stable stable state, and add another 60s for avocado script's running